rosbag2_tests/test/rosbag2_tests/record_fixture.hpp

```diff
@@ -45,17 +45,13 @@ class RecordFixture : public TemporaryDirectoryFixture
 public:
   RecordFixture()
   {
-    root_bag_path_ = (rcpputils::fs::path(temporary_dir_path_) / "bag").string();
-    storage_path_ = (rcpputils::fs::path(root_bag_path_) / "bag_0").string();
-    database_path_ = storage_path_ + ".db3";
-    std::cout << "Database " << database_path_ << " in " << temporary_dir_path_ << std::endl;
+    root_bag_path_ = rcpputils::fs::path(temporary_dir_path_) / "bag";
   }
 
   void SetUp() override
   {
-    const auto path = rcpputils::fs::path{root_bag_path_};
-    if (rcpputils::fs::exists(path)) {
-      remove_directory_recursively(path.string());
+    if (root_bag_path_.exists()) {
+      remove_directory_recursively(root_bag_path_.string());
```

Comment:
Nice - what about adding a comment explaining the rationale here?
E.g.

Clean-up potential leftover bag files. If a previous run of this script crashed, it may have leaked test bags to the disk. We are cleaning up the directory structure before starting the test to avoid interferences due to previous test runs.

Reply:
Done